### ISSUE TITLE
Error during initial_pose_estimation.

### ISSUE THREAD

**Issue by shuluoshu** [yshah43/DPOD#12]
Hi @yshah43  
Thanks for your work. I met a problem in **pose_block.py** in `coord_2d = (temp == classes[label]).nonzero(as_tuple=True)` , Since after training 20 epoch and inference, the returned coord_2d is of shape (0, 2). In other words, there is no `temp == classes[label]` results. The training loss is small. So I wonder what's the problem. Could you offer me a pretrained model of this step "train_correspondence_block(root_dir, classes, epochs=20)".
Thank you very much!
